ieeg_epilepsy/sub-01/ses-postimp/ieeg/sub-01_ses-postimp_space-MNI152_electrodes.tsv

```diff
@@ -0,0 +1,125 @@
+name	x	y	z	size	hemisphere	group	type	manufacturer
+v'1	-1.3	-39	31	5	L	v'	depth	DIXI
```

Comment:
name: Label of the channel, only contains letters and numbers.

Reply:
mmh this might be complicated if in the raw data the channels have other symbols in their label ... do you have a suggestion for a fix that would not involve reading in all data and renaming the labels?